packages/block-library/src/image/image.js

```diff
@@ -689,8 +697,8 @@ export default function Image( {
 					onResizeStop();
 					setAttributes( {
 						width: elt.offsetWidth,
-						height: elt.offsetHeight,
-						aspectRatio: undefined,
+						height: 'auto',
```

Comment:
This is the main change. We are now relying on the width and aspect ratio to infer the height.